pkg/server/node.go

```diff
@@ -195,6 +195,30 @@ This metric is thus not an indicator of KV health.`,
 		Measurement: "Bytes",
 		Unit:        metric.Unit_BYTES,
 	}
+	metaActiveRangeFeed = metric.Metadata{
+		Name:        "rpc.streams.rangefeed.count",
```

Comment:
Count seems like a misnomer, since this is a gauge not a counter. Active? Current?

Reply:
Ack.